### DISCUSSION TITLE
Changing smiley background colour in tmux

### DISCUSSION THREAD

**Discussion by matko031** [dracula/dracula-theme#817] [Q&A] [answered]
Is it possible to configure the background colour of the smiley section in the tmux theme? I sometimes have to use nested tmux sessions and having different colour there would make it much easier to quickly see in which environment am I working.

**Comment by Chucksec** [comment, marked as answer]
Yes, it is possible to configure the background color of the smiley section in the tmux theme. You can use the following command in your tmux configuration file (usually located at ~/.tmux.conf) to set the background color of the section:

`
set -g status-style bg=<color>`

Replace <color> with the color code you want to use for the background. For example, if you want to use a red background, you can use:


`set -g status-style bg=red`

This will set the background color of the entire status line, including the smiley section. Using different colors for nested tmux sessions can help you quickly identify which environment you are working in.
